Gemfile.lock

```diff
@@ -8,10 +8,10 @@ GIT
 
 GIT
   remote: https://github.com/mconf/bigbluebutton-api-ruby.git
-  revision: 91dc495324a6b7e162773227ec3650f8a5b39c50
+  revision: 803c1e0680eada08740bb068956647bdd36ac7ee
```

Comment:
There is a problem with commit https://github.com/mconf/bigbluebutton-api-ruby/commit/803c1e0680eada08740bb068956647bdd36ac7ee

It should be 5b803bd640e5144d32b18e039e2bc63c85c0bb4f

Reply:
Yes, that's bizarre It's bundler who updated the `bigbluebutton-api-ruby` dependency.
On the time of creating the PR that was the gem remote revision and the CI pipeline succeeded.
There must be something with their configuration somehow `wpramio` fork got caught as a valid remote revision for the gem.
 @farhatahmad we have to be aware of this since it can happen again and could you check with mconf about it?
Meanwhile I'll reupdate the dependency and make sure that it points to a valid commit on their master branch.

Reply:
https://github.com/KH-Amir-TN/greenlight/runs/4827195801?check_suite_focus=true#step:9:569
But 5 hours ago after @farhatahmad merged on my forked repo which in turn launched the CI pipeline again failed indicating that **803c1e0** cannot be found on master branch.
https://github.com/KH-Amir-TN/greenlight/runs/4876692555?check_suite_focus=true#step:9:349